certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -561,3226 +649,8098 @@
+            "type": "string",
+            "default": "1",
+            "x-ms-visibility": "internal",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "phoneNumber",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Phone Number.",
+            "x-ms-summary": "Phone Number"
+          },
+          {
+            "name": "includeAddress",
+            "in": "query",
+            "required": true,
+            "type": "boolean",
+            "default": true,
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "includePhone",
+            "in": "query",
+            "required": true,
+            "type": "boolean",
+            "default": true,
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "includeEmail",
+            "in": "query",
+            "required": true,
+            "type": "boolean",
+            "default": true,
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/api/v1/Events/Registrants/{recordNumber}/Attended": {
+      "put": {
+        "tags": [
+          "Events"
+        ],
+        "summary": "Mark Registrant as Attended",
+        "description": "Mark the Registrant as Attended for the specified Events.",
+        "operationId": "Mark-Registrant-Attended",
+        "parameters": [
+          {
+            "name": "recordNumber",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Record Number.",
+            "x-ms-summary": "Record Number",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "x-ms-summary": "Data List",
+            "required": true,
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "eventOrSessionCode": {
+                    "type": "string",
+                    "description": "Event or Session Code.",
+                    "title": "Code"
+                  }
+                },
+                "required": [
+                  "eventOrSessionCode"
+                ]
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
```

Comment:
@RjImpx Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.